igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisService.groovy

```diff
@@ -220,8 +236,34 @@ class TravisService implements BuildService {
         return null
     }
 
+    boolean hasRepo(String inputRepoSlug) {
+        String repoSlug = cleanRepoSlug(inputRepoSlug)
+        try {
+            getRepo(repoSlug)
+        } catch (RetrofitError error) {
+            if (error.getResponse()) {
+                if (error.getResponse().status == 404) {
+                    log.debug "repo not found ${repoSlug}"
+                    return false
+                }
+            }
+            log.info "Error requesting repo ${repoSlug}: ${error.message}"
+            return true
+        }
+        return true
+    }
+
     @Override
     BuildServiceProvider buildServiceProvider() {
         return BuildServiceProvider.TRAVIS
     }
+
+    protected String cleanRepoSlug(String inputRepoSlug) {
+        def parts = inputRepoSlug.tokenize('/')
+        return "${parts[0]}/${parts[1]}"
```

Comment:
Im not sure, I think all github and stash repos has org/repo but I am not 100% sure.